### PR TITLE
7.x

### PR DESCRIPTION
**Pull request by rosiel**
Default sort feature - as discussed at 2013-03-21 meeting

### CODE REVIEW

File: includes/query_processor.inc [hunk 1]
```diff
@@ -326,7 +335,7 @@ class IslandoraSolrQueryProcessor {
           // Hook to alter everything.
           drupal_alter('islandora_solr_results', $object_results, $this);
           // Additional Solr doc preparation. Includes field permissions and limitations.
-          $object_results = $this->prepareSolrDoc($object_results);
+          $object_results = $this->prepareSolrDoc($object_results, $solr_results['highlighting']);
```
Comment (DannyJoris) [inline includes/query_processor.inc]: Highlighting got moved away from ::prepareSolrDoc in a previous commit.

File: islandora_solr.install [hunk 2]
```diff
@@ -75,7 +75,9 @@ function islandora_solr_uninstall() {
     'islandora_solr_limit_result_fields',
     'islandora_solr_num_of_results',
     'islandora_solr_namespace_restriction',
+    'islandora_solr_base_sort',
     'islandora_solr_base_query',
+    'islandora_solr_base_sort',
```
Comment (DannyJoris) [inline islandora_solr.install:79]: duplicate?

**Comment by rosiel** [comment]
Thanks for your help, Danny! It was an identity crisis - I confused "mine" with "theirs" in the merge.

**Comment by DannyJoris** [comment]
No worries - that happens. Thanks for contributing!

**Comment by DannyJoris** [comment]
hah.

**Pull request merged**